### ISSUE TITLE
[Bug]: Multiple "[object Object]" seen in log files from LibraryItem.js:386

### ISSUE THREAD

**Issue by ZLoth** [advplyr/audiobookshelf#3362]
### What happened?

In the log files, I am seeing multiple [object Object] when importing or updated an item. 

### What did you expect to happen?

Actual data for diagnosing or reporting issue. 

### Steps to reproduce the issue

Import or update an audiobook, then review the log files. 

### Audiobookshelf version

v2.12.3

### How are you running audiobookshelf?

Docker

### What OS is your Audiobookshelf server hosted from?

Linux

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

None

### Logs

```shell
{"timestamp":"2024-09-02 06:40:57.506","source":"LibraryItem.js:386","message":"[LibraryItem] \"Flying the SR-71 Blackbird\" libraryFiles updated from [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object] to [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]","levelName":"DEBUG","level":1}
{"timestamp":"2024-09-02 06:41:17.472","source":"LibraryItem.js:369","message":"[LibraryItem] \"Flying the SR-71 Blackbird\" book.chapters updated from [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object] to [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]","levelName":"DEBUG","level":1}
{"timestamp":"2024-09-02 06:41:17.575","source":"LibraryItem.js:386","message":"[LibraryItem] \"Flying the SR-71 Blackbird\" libraryFiles updated from [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object] to [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]","levelName":"DEBUG","level":1}
```

### Additional Notes

Not applicable.

**Comment by advplyr** [comment]
The libraryFiles log was updated in v2.13.0. Is that log from an older server version?

**Comment by advplyr** [comment]
I misread the server version you specified in the issue. The libraryFiles log was fixed in v2.13.0. The chapters log still isn't fixed.
